src/components/dashboard/setup-profile-tab/SetupProfileTab.vue

```diff
@@ -29,13 +29,15 @@
         <p>Zebras communicate with facial expressions and sounds. They make loud braying or barking sounds and
           soft snorting sounds. The position of their ears, how wide open their eyes are, and whether they show
           their teeth all send a signal. For example, ears flat back means trouble, or "you better follow orders!"</p>
+
         <vuestic-simple-select
           label="Select country"
           v-model="selectedCountry"
           name="country"
           :required="true"
           ref="selectedCountrySelect"
-          v-bind:options="countriesList">
+          :options="countriesList"
+          :tempOptions="countriesArray">
```

Comment:
What is that?

Reply:
I use tempOptions to store sorted list. For example when we want to find countries starting on 'A' all that countries gonna store in tempOptions